guides/v2.3/javascript-dev-guide/javascript/requirejs.md

```diff
@@ -0,0 +1,105 @@
+---
+group: javascript-developer-guide
```

Comment:
This guide is applicable for both 2.2 and 2.3. Please, make the physical copy for v 2.2 and a symlink for 2.3

Reply:
Have symlinked it, thanks.